### PR TITLE
Added a PR template.

### PR DESCRIPTION
**Pull request by gyliu513**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
+Fixes #
+
+**Special notes for your reviewer**:
+
+1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.
```
Comment (wking) [inline .github/PULL_REQUEST_TEMPLATE.md]: I don't understand "changes any image versions".  Can you rephrase?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+**Release note**:
+<!--  Write your release note:
+1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
+2. If no release note is required, just write "NONE".
```
Comment (wking) [inline .github/PULL_REQUEST_TEMPLATE.md]: We don't have tooling to scrape these.  Changes that require user changes should instead be documented in a Bugzilla linked from the PR subject with a `Bug ###` prefix.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (gyliu513) [reply .github/PULL_REQUEST_TEMPLATE.md]: Sorry, I got this template from https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/master/.github/PULL_REQUEST_TEMPLATE.md , but seems this sentence needs to be removed.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (gyliu513) [reply .github/PULL_REQUEST_TEMPLATE.md]: But I can see some people also opening issues in Github but not only Bugzilla, can we use https://github.com/github-changelog-generator/github-changelog-generator to generate kind of CHANGELOG?

**Comment by patrickdillon** [comment]
I agree with https://github.com/openshift/installer/pull/2414#pullrequestreview-293748984

Seems like this is trying to solve a problem we don't have and I think it might get in the way.  I do like the idea of a template in principle but seems like this would result in my needing to delete text from the template for each pr (more work/mental overhead).

**Comment by gyliu513** [comment]
thanks @patrickdillon and @wking , let me close this one.
